### PR TITLE
fix(settings): should be wrapped in "intelephense"

### PR DESCRIPTION
**Pull request by jfcherng**
https://github.com/sublimelsp/LSP-intelephense/issues/22#issuecomment-605525589

### CODE REVIEW

File: LSP-intelephense.sublime-settings [hunk 1]
```diff
@@ -1,24 +1,26 @@
 {
-	"languages": [
-		{
-			"languageId": "php",
-			"scopes": ["source.php"],
-			"syntaxes": [
-				"Packages/PHP/PHP.sublime-syntax",
-			],
+	"intelephense": {
```
Comment (rchl) [inline LSP-intelephense.sublime-settings]: But why? The plugin code doesn't read it from that key as far as I can see.

Comment (rchl) [reply LSP-intelephense.sublime-settings]: I've looked at the issue and I get what you were trying to do here but I think you meant to put that key within the `settings` object.

**Comment by rchl** [comment]
While you are at it, you could change
```
        return read_client_config("lsp-intelephense", default_configuration)
```
to
```
        return read_client_config(self.name, default_configuration)
```
maybe? :)
(just a style change)

File: LSP-intelephense.sublime-settings (hunk 1 above)
Comment (jfcherng) [reply LSP-intelephense.sublime-settings]: Ah, yes!

**Pull request merged**